packages/replayer/src/replayer.ts

```diff
@@ -121,14 +126,14 @@ export const replayEvents: ReplayEventsFn = async (options) => {
         page: Page;
         logLevel: LogLevelDesc;
         sessionData: SessionData;
-        startUrl: string;
+        originalSessionStartUrl: string;
         onTimelineEvent: OnReplayTimelineEventFn;
       }) => Promise<void>;
     await setupReplayNetworkStubbing({
       page,
       logLevel,
       sessionData,
-      startUrl,
+      originalSessionStartUrl: originalSessionStartUrl.toString(),
```

Comment:
For back compat I'll make the meticulous side do `originalSessionStartUrl ?? startUrl // CLI versions prior to https://github.com/alwaysmeticulous/meticulous-sdk/pull/151/files send through a startUrl instead`.

Reply:
Actually realized I do still need the resolved startUrl. I'll pass both.